CONTRIBUTING.md

```diff
@@ -7,20 +7,19 @@ at [the development process documentation](https://docs.opencast.org/develop/dev
 guide should suffice for most contributions.
 
 
-Bug Fixes and Feature
----------------------
+Accepting Criteria for Patches in Different Versions
```

Comment:
Does it make sense to have this be a link to the development process?  Otherwise we'll (eventually) see version drift...

Reply:
While I would usually agree that we should not duplicate things, just linking to the development process (which is already linked to above) kind of defeats the purpose of this file which people can read as a quick-guide to get the most important information they need to know nefore contributing without reading through the lengthy development process documentation. That's why I would like to leave this shortened version of the development process section in here.